.github/workflows/lints.yaml

```diff
@@ -9,10 +14,8 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3.5.0
```

Comment:
It's kinda mess to have everything as one step. I suggest 4 steps:
- checkout
- install dependencies
- run pylint
- run bandit

This way you will have nice breakdown when you run it and it will be much more clean from the first glance what when wrong.

See example here: https://github.com/ispc/ispc/blob/main/.github/workflows/format-check.yml

Reply:
Agreed & addressed in the branch

Reply:
I don't think so. They are not dependencies, so their installation logically belongs to corresponding steps, it seems

Reply:
ok